src/ape/pytest/runners.py

```diff
@@ -1,101 +1,35 @@
-from pathlib import Path
-
-import click
 import pytest
 from _pytest.config import Config as PytestConfig
 
 import ape
 from ape.logging import logger
-from ape.managers.chain import ChainManager
-from ape.managers.networks import NetworkManager
-from ape.managers.project import ProjectManager
-from ape_console._cli import console
+from ape.utils import ManagerAccessBase
 
 from .contextmanagers import RevertsContextManager
 
 
-class PytestApeRunner:
-    def __init__(
-        self,
-        config: PytestConfig,
-        project: ProjectManager,
-        networks: NetworkManager,
-        chain: ChainManager,
-    ):
-        self.config = config
-        self.project = project
-        self.networks = networks
-        self.chain = chain
+class PytestApeRunner(ManagerAccessBase):
+    def __init__(self, pytest_config: PytestConfig):
+
+        self.pytest_config = pytest_config
         self._warned_for_missing_features = False
         ape.reverts = RevertsContextManager  # type: ignore
 
     @property
     def _network_choice(self) -> str:
-        # The option the user providers via --network (or the default).
-        return self.config.getoption("network")
-
-    def pytest_exception_interact(self, report, call):
```

Comment:
Looks like we accidentally deleted all of the console related stuff?

- [ ] Make sure `-I` still works for `ape test`